layouts/shortcodes/observability_pipelines/processors/custom_processor.md

```diff
@@ -27,4 +29,5 @@ To set up this processor:
 1. Click **Save**.
 
 [10191]: /observability_pipelines/processors/custom_processor#custom-functions
-[10192]: /observability_pipelines/guide/get_started_with_the_custom_processor
\ No newline at end of file
+[10192]: /observability_pipelines/guide/get_started_with_the_custom_processor
+[10193]: /observability_pipelines/processors/edit_fields
```

Comment:
Is this the intended link? I thought it might go to the new guide.

Reply:
Thanks, good catch! Updating:
```suggestion
[10193]: /observability_pipelines/guide/remap_reserved_attributes
```